<?xml version="1.0"?>
<launch>

	<!--<include file="$(find pr2_controller_manager)/controller_manager.launch"/>-->
	<!--<param name="robot_description" command="'$(find pr2_description)/robots/youbot.urdf.xacro'" />-->

	<!-- Controller manager parameters -->
	<!--<param name="pr2_controller_manager/mechanism_statistics_publish_rate" value="1.0" />-->
	<!--<param name="pr2_controller_manager/joint_state_publish_rate" value="100.0" />-->

	<!-- Diagnostics -->
	<!--<node pkg="pr2_mechanism_diagnostics" type="pr2_mechanism_diagnostics" name="pr2_mechanism_diagnostics" />-->
        
	<!-- Gripper Control -->
        <rosparam command="load" file="$(find youbot_description)/controller/gripper_joint_position_control.yaml"  ns="arm_1"/>
	
	<!-- Arm Control -->
        <rosparam command="load" file="$(find youbot_description)/controller/arm_joint_universal_control.yaml"  ns="arm_1"/>
	
         <!-- base control -->
  	<rosparam command="load" file="$(find youbot_description)/controller/base_velocity_control.yaml"  />

  	<!-- base odometry -->
  	<rosparam file="$(find youbot_description)/controller/base_odometry.yaml" command="load" /> 

	
	<!-- Nodes on top of the controllers -->        

<!--  Removed because moveit is not currently being used to control grippper-->
<!--
  	<node name="gripper_controller_spawner" pkg="pr2_controller_manager" type="spawner" args="arm_1/gripper_controller" />
-->

  	<node name="arm_controller_spawner" pkg="pr2_controller_manager" type="spawner" args="arm_1/arm_controller"/>

  <!-- Removed because moveit is not currently being used to control base-->
<!--
  	<node name="base_controllers_spawner" pkg="pr2_controller_manager" type="spawner" output="screen" args="base_controller base_odometry" /> 
-->
</launch>
